src/components/vr-scatter-plot/vr-scatter-plot.component.ts

```diff
@@ -9,16 +9,21 @@ import { LineChartMeta } from '../../datasets/metas/line-chart.meta';
 import { BehaviorSubject, Subject } from 'rxjs';
 import { DAY } from '../../utils/timeUnits';
 import { map, takeUntil } from 'rxjs/operators';
+import { DATA_PREFERENCE } from '../../i18n';
+import { datasets } from '../../datasets';
 
 @Component({
   selector: 'app-vr-scatter-plot',
-  templateUrl: './vr-scatter-plot.component.html'
+  templateUrl: './vr-scatter-plot.component.html',
+  styleUrls: ['./vr-scatter-plot.component.scss'],
 })
 export class VRScatterPlotComponent<T extends Meta>implements OnInit, OnChanges, OnDestroy{
   @Input() endDate = new Date();
   @Input() startDate = new Date(this.endDate.getTime() - 30 * DAY);
-  @Input() meta: Meta;
+  @Input() meta: Meta | Meta[];
   @Input() meta2: LineChartMeta;
```

Comment:
Similar to mfitz's comment earlier, a more descriptive name here will be helpful.

Reply:
renamed `meta` to `datasetPref`